src/affiliations/urls.py

```diff
@@ -9,7 +9,7 @@
 from affiliations import views
 
 urlpatterns: list[URLResolver | URLPattern] = [
-    path("", views.AffiliationsList.as_view()),
+    path("'", views.AffiliationsList.as_view()),
```

Comment:
I'm curious about this. Is this a single quote?

Reply:
This was an error/mistake, this has been remedied.